src/LockstakeEngine.sol

```diff
@@ -219,6 +219,7 @@ contract LockstakeEngine is Multicall {
     // --- getters ---
 
     function getUrn(address owner, uint256 index) external view returns (address urn) {
+        // NOTE: this function will succeed returning the address even if the urn for the specified index hasn't been created yet
```

Comment:
Nit - I'd add this comment above the function and not inside.

Reply:
Done.